app/api/dao/user.py

```diff
@@ -165,6 +167,16 @@ def list_users(user_id, search_query="", is_verified=None):
             )
         ]
 
+        filter_by_name = lambda list_of_users: list_of_users['name']
+
+        if sort_order!='':
```

Comment:
Don't need this check

Reply:
How about we implement this with help of https://flask-restless.readthedocs.io/en/stable/searchformat.html. 
1. This way can follow same conventions everywhere. 
1. We dont have to write the filtering logic by ourself